react/components/RequestOrganizationForm.tsx

```diff
@@ -33,6 +33,7 @@ import { getEmptyAddress, isValidAddress } from '../utils/addresses'
 import CREATE_ORGANIZATION_REQUEST from '../graphql/createOrganizationRequest.graphql'
 import GET_ORGANIZATION_REQUEST from '../graphql/getOrganizationRequest.graphql'
 import GET_LOGISTICS from '../graphql/getLogistics.graphql'
+import CheckCustomerSchema from './CheckCustomerSchema'
```

Comment:
I don't think we should show this message in the frontend. The store admin is the only one who can fix the problem, so we should only show in the admin panel.